recipes/r-cleangeostreamr/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "cleangeostreamr" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: r-{{ name }}
+  version: {{ version }}
+
+source:
+  url: "https://codebase.helmholtz.cloud/department-computational-biology/software/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz"
+  sha256: d1d9f5f0e79527cde6aeb88535cf17d38966af0ce1db2108e6cf9c8aad33d18d
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  run_exports:
+    - {{ pin_subpackage("r-cleangeostreamr", max_pin="x.x") }}
+  script: |
+    R CMD INSTALL --build .
+
+requirements:
+  host:
+    - r-base
+    - r-devtools
+    - r-dt
+    - r-tidyverse
+    - r-optparse
+    - r-lubridate
+    - r-tidygeocoder
+    - r-readr 
+    - r-r.utils
+    - r-rnaturalearth
+    - r-rnaturalearthdata
+    - r-sf
+    - r-sp
+    - r-leaflet
+    - r-jsonlite
+    - r-magrittr
+    - r-stringi
+    - r-dplyr
+    - r-ggplot2
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+  run:
+    - r-base
+    - r-devtools
+    - r-dt
+    - r-tidyverse
+    - r-optparse
+    - r-lubridate
+    - r-tidygeocoder
+    - r-readr 
+    - r-r.utils
+    - r-rnaturalearth
+    - r-rnaturalearthdata
+    - r-sf
+    - r-sp
+    - r-leaflet
+    - r-jsonlite
+    - r-magrittr
+    - r-stringi
+    - r-dplyr
+    - r-ggplot2
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+
+test:
+  commands:
+    - $R -e "library('CleanGeoStreamR')"
```

Comment:
Last but not least can you add a test for the executable scripts, e.g. using the `--help` argument?